### PR TITLE
fix broken links in kubeslice-cli docs

### PR DESCRIPTION
**Pull request by uma-kt**

### CODE REVIEW

**Comment by uma-kt** [comment]
> The sample topology link also needs to be fixed in https://kubeslice.io/documentation/open-source/0.4.0/kubeslice-cli/installing-kubeslice

Fixed all the broken links.

File: versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-commands/register.mdx [hunk 1]
```diff
@@ -21,15 +21,15 @@ The following are the `kubeslice-cli register` command options.
...
+[Kubeslice Configuration Parameters](/versioned_docs/version-0.4.0/reference/configuration-parameters.mdx). 
 
 ## Examples
 The following are the example commands:
 :::info
 You must switch the context to the controller cluster to register the worker cluster.
 :::
 
-1. To register a worker cluster with the KubeSlice Controller, use the following command:
+1. To register a worker cluster with the KubeSlice Controller in demo set up, use the following command:
```
Comment (veenamj-avesha) [inline versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-commands/register.mdx:32]: set up is action 

here it is `in the demo setup `

**Review by veenamj-avesha — APPROVED** [review]
LGTM

File: versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-commands/register.mdx (hunk 1 above)
Comment (uma-kt) [reply versioned_docs/version-0.4.0/kubeslice-cli/kubeslice-cli-commands/register.mdx]: updated

**Pull request merged**